### DISCUSSION TITLE
how can I use tools in llm node of workflow?

### DISCUSSION THREAD

**Discussion by hh1562535601** [langgenius/dify#12197] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I use workflow to build my app, and I need use function calling with llm node, but now llm node does not support this, and I know agent supports tools, so what should I do to solve this problem?Thanks!

**Comment by JerryYao75** [comment]
I met this problem either.
